platform/ui/src/views/StudyList/StudyList.js

```diff
@@ -1,18 +1,55 @@
 import React from 'react';
+import PropTypes from 'prop-types';
+
 import Header from './components/Header';
 import StudyListFilter from './components/StudyListFilter';
 import StudyListTable from './components/StudyListTable';
 import StudyListPagination from './components/StudyListPagination';
 
-const StudyList = () => {
+import studyListMock from '../../mocks/studyList.json';
+
+/** Values can be env vars */
+const DEFAULT_MOCKED_STUDIES_NUM = 50;
+const DEFAULT_MOCKED_STUDIES_LIMIT = 1000;
```

Comment:
Don't bake this into the component. This would happen in docz or somewhere similar. Potentially with throw-away ui to change data. Might be worth looking at React powerplug?

Reply:
Hmm.. Unless we are considering this top level container throwaway. Worth discussing. But baking this in makes copy-pasting general structure over slightly more difficult

Reply:
I have changed this logic. Now the mocking data logic is only tight to Docz.